epub33/core/index.html

```diff
@@ -5516,8 +5516,8 @@ <h4>Path and File Names</h4>
 							</ul>
 						</li>
 						<li>
-							<p id="ocf-fn-cn">All File Names within the same directory MUST be unique following full
-								case folding [[Unicode]] and NFC normalization [[TR15]]. (Refer to <a
+							<p id="ocf-fn-cn">All File Names within the same directory MUST be unique following 
+								NFC normalization [[TR15]] and then case folding [[Unicode]]. (Refer to <a
```

Comment:
I believe we still need to say "full case folding", no, otherwise isn't there ambiguity with simple case folding?

Reply:
Damn, my mistake. Thanks, done!